src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -277,7 +280,8 @@ const formConfig = {
               'ui:options': {
                 showFieldLabel: 'label',
                 label: disabilityOption,
-                widgetClassNames: 'widget-outline'
+                widgetClassNames: 'widget-outline',
+                keepInPageOnReview: true
```

Comment:
@jbalboni `keepInPageOnReview` is now used here, per your helpful recommendation.